### ISSUE TITLE
[FEATURE REQUEST]Please support Linux.

### ISSUE THREAD

**Issue by arabcian** [JamesCJ60/Universal-x86-Tuning-Utility#242]
Tried to build PR you did on ryzenadj with experimental 7045hx series support with no luck it just hangs for some time and then errors out. I have a 7845hx and want to tweak it under Linux is there any possibility of Linux support in the future?

**Comment by Vistaus** [comment]
I would also like to see that as I am a Linux user.

**Comment by rabiddoughnuts** [comment]
I would also love to see this, ryzenadj or amdctl both "work" but this tool is so much easier and intuitive to use imo

**Comment by Alastor-Ripley** [comment]
I would also love to see a linux version

**Comment by JamesCJ60** [comment]
Hi all, I cannot get into much detail right now, but UXTU’s successor, which I have started development work on, aims to support Linux (provided nothing goes wrong) and will merge a few of my projects into one (e.g., UXTU, UXTU Handheld, Framework Hub, etc.).

**Comment by arabcian** [comment]
Woah thats awesome to hear.

**Comment by officiallymarky** [comment]
I'd love to see this on Linux, or an update on your new version.

**Comment by JamesCJ60** [comment]
Hi all! Just wanted to keep you all in the loop. @loyvsc is currently working on a Linux port of this version of UXTU. It’s a work in progress, so we appreciate your patience and support. Again, big thanks to @loyvsc for taking this on!

**Comment by arabcian** [comment]
Hello Guys Good news that JamesCJ's pull request on ryzenadj about DragonRange and FireRange preliminary support has accepted, now i can use curve optimizer on my 7845hx

**Comment by Danielli64** [comment]
Yes!! Just tested RyzenAdj and set the Curve Optimizer to -25 on my 7945HX in Linux (Debian 13) and it bl..dy worked. :) Ran passmark and performance numbers i have never seen before. Nice!!!!

**Comment by Danielli64** [comment]
> Hello Guys Good news that JamesCJ's pull request on ryzenadj about DragonRange and FireRange preliminary support has accepted, now i can use curve optimizer on my 7845hx

Thanks for the heads up. I would never have seen otherwise. It would be nice if they could add min and max CPU frequency. One core stops att -25 CO all others can run att -30 or less. Reducing the CPU frequency from 5450 to 5400 MHz the last core can also goes to -30. Even better, set frequency per core :) Tested this on Windows with SMUDebugTool where it is possible to set global frequency but not per core. You always want more more feature. :)

**Comment by JamesCJ60** [comment]
Please see an update regarding this here: https://github.com/JamesCJ60/Universal-x86-Tuning-Utility/issues/179#issuecomment-4763279115
